.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
   compile:
     runs-on: ubuntu-20.04
-    timeout-minutes: 60
+    timeout-minutes: 120
```

Comment:
Is this necessary?

Reply:
I should take < 60, but now we're close to the deadline, and I wouldn't want to fail a whole build just because this step took 5 mins longer on a busy day...

Reply:
Okay how much of this is being driven by Scala Native? Four days ago we were taking a little over 30 minutes.